share/power-bi/README.md

```diff
@@ -0,0 +1,82 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+  </a>
+</p>
+
+# Reading from a Uni with Microsoft Power BI
+Vendia Share makes sharing data easy and provides a platform upon which applications can be built using a zero-code GraphQL API.  The power and flexibility of an API based on your data model is amazing, but what if you haven't used graphql before? What if you don't have access to software developers to build atop of the API? Fortunately there are a number of data visualization and reporting applications that work very well with Uni data. This example demonstrate how to use Microsoft Power BI with a Vendia Share Uni to create reports and visualizations of data read from the Uni's GraphQL API.
+
+## How It Works
+Data visualization and reporting tools leverage common interfaces, like APIs, to retrieve data. These applications often provide mechanisms to transform data and familiar reporting controls like charts and tables.  In this example, Power BI is connected to a Uni using  M language (Power Query Formula Language) to query the Uni's GraphQL API.  Additional M language functions then transform the JSON data returned by the API into a data table.  Once in table form, Power BI tools can be used to create tables, charts, and more for visualizing and sharing Uni data with the Power BI Platform.
+
+## Pre-requisites
+
+**Exiting Uni**
+This example assumes that you already have a functioning Uni with data.  Please refer to our [Quick Start Documentation](https://vendia.net/docs/share/quickstart) to learn how to quickly create a Uni. If you would like to follow along using queries from this example, you may create a uni with the matching [Example Schema](example_schema.json).  
+
+**Important** Once your Uni is Running, add some data using the addInventoryItem_async mutation. Otherwise, there will be no data to visualize in later steps.
```

Comment:
Can we populate the uni with sample data?  Might be helpful to users just getting started.